Framework/Kernel/inc/MantidKernel/DateAndTime.h

```diff
@@ -114,30 +114,24 @@ class MANTID_KERNEL_DLL DateAndTime {
   static void createVector(const DateAndTime start,
                            const std::vector<double> &seconds,
                            std::vector<DateAndTime> &out);
-  static bool stringIsISO8601(const std::string &str);
+
+  /// The difference in seconds between standard unix and gps epochs.
+  static const uint32_t EPOCH_DIFF = 631152000;
+
+  /// The epoch for GPS times.
+  static const boost::posix_time::ptime GPS_EPOCH;
+
+  /// Const of one second time duration
+  static const time_duration oneSecond;
+
+  static time_t utc_mktime(struct tm *utctime);
```

Comment:
Should this in fact be in the helpers namespace?

Reply:
It is used internally (within `DateAndTime`) as well.